README.md

```diff
@@ -38,166 +14,6 @@ Rafiki also allows for delegated access, offering OAuth-based flows to grant
 third-party applications access to Interledger functionality on a user's
 account.
 
```

Comment:
A suggestion: Should we include links to the Interledger docs or something that introduces people to the different features of Interledger that Rafiki will expose? I like what they did here https://github.com/mozilla/glean.js

Reply:
I've added some general links to NB parts of the ecosystem. Maybe we should go over this part and flesh it out a bit more.